app/assets/stylesheets/spinners.scss

```diff
@@ -0,0 +1,78 @@
+
+/*
+  Spinner directive
+*/
+us-spinner {
+  position: relative;
+  display: block;
+
+  [role="progressbar"]::after {
+    content: 'Loading...';
+    position: absolute;
+    font-size: 14px;
+    width: 150px;
+    text-align: center;
+    top: 30px;
+    left: -70px;
+  }
+}
+
+/*
+  Spin animation (triggered through CSS class)
+*/
+.spinning {
+  -webkit-animation-name: spin;
+  -webkit-animation-duration: 4000ms;
+  -webkit-animation-iteration-count: infinite;
+  -webkit-animation-timing-function: linear;
+  -moz-animation-name: spin;
+  -moz-animation-duration: 4000ms;
+  -moz-animation-iteration-count: infinite;
+  -moz-animation-timing-function: linear;
+  -ms-animation-name: spin;
+  -ms-animation-duration: 4000ms;
+  -ms-animation-iteration-count: infinite;
+  -ms-animation-timing-function: linear;
+  animation-name: spin;
+  animation-duration: 4000ms;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+
+@-ms-keyframes spin {
+  from {
+    -ms-transform: rotate(0deg);
+  }
+  to {
+    -ms-transform: rotate(360deg);
+  }
+}
+
+@-moz-keyframes spin {
+  from {
+    -moz-transform: rotate(0deg);
+  }
+  to {
+    -moz-transform: rotate(360deg);
+  }
+}
+
+@-webkit-keyframes spin {
+  from {
+    -webkit-transform: rotate(0deg);
+  }
+  to {
+    -webkit-transform: rotate(360deg);
+  }
+}
+
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+
```

Comment:
This is the spinning animation I'm using for the typeahead button. When you type something in the typeahead and then you click on the button to submit the reference, the button gets replaced by a icon that spins while the API call is ongoing. All this is done through CSS, so, to animate it, I need this CSS.

The spinner gem is being used for a different purpose: it offers an angular directive to display a pre-defined spinner that you can show as a modal (default) or in place of the content while the content is not rendered (that's what I'm doing).

But because both have spinning animations, I refactored it to this spinners.scss file.